system-parachains/bridge-hubs/bridge-hub-kusama/tests/tests.rs

```diff
@@ -326,6 +328,21 @@ pub fn can_calculate_weight_for_paid_export_message_with_reserve_transfer() {
 	)
 }
 
+#[test]
+fn change_ethereum_gateway_by_governance_works() {
```

Comment:
@claravanstaden very nit, why not to move it to the `tests/snowbridge.rs` also for BHP

Reply:
https://github.com/polkadot-fellows/runtimes/pull/130/commits/756d00b6acb67908e866a258e2cc45c792ade930